src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/KinesisClientLibLeaseCoordinator.java

```diff
@@ -130,6 +129,16 @@ public KinesisClientLibLeaseCoordinator(ILeaseManager<KinesisClientLease> leaseM
         this.leaseManager = leaseManager;
     }
 
+    public KinesisClientLibLeaseCoordinator(ILeaseManager<KinesisClientLease> leaseManager,
```

Comment:
All of the rest of the constructors have workeridentifier, but not this one. Is there a reason for it? LeaseCoordinator should require worker identifier.

Reply:
WorkerIdentifier is a local variable used for initialization of LeaseTaker. Since LeaseTaker is being injected, we don't need to pass in workerIdentifier separately.